pbp.sh

```diff
@@ -1,27 +1,20 @@
-echo "$(tput setaf 5)Pulling and bundling consumer-layouts...$(tput sgr0)"
-cd consumer-layouts
-git pull
-bundle install
-
-echo "$(tput setaf 5)Pulling and bundling consumer-domain...$(tput sgr0)"
-cd ../consumer-domain
-git pull
-bundle install
-
-echo "$(tput setaf 5)Pulling and bundling consumer-session...$(tput sgr0)"
-cd ../consumer-session
-git pull
-bundle install
-
-echo "$(tput setaf 5)Pulling and bundling domain-values...$(tput sgr0)"
-cd ../domain-values
-git pull
-bundle install
-
-echo "$(tput setaf 5)Pulling and bundling consumer-main...$(tput sgr0)"
-cd ../consumer-main
-git pull
-bundle install
+#!/usr/bin/env bash
+
+REPOS="consumer-layouts
+consumer-domain
+consumer-session
+domain-values
+consumer-main"
+
+ROOT=$(pwd)
+
+for dir in $REPOS
+do
+  echo "$(tput setaf 5)Pulling and bundling ${dir}...$(tput sgr0)"
+  cd "${ROOT}/${dir}"
```

Comment:
`$(pwd)` runs the pwd command in every iteration of the loop. For example, `/Users/cewen/code` is my directory. First time, it gets consumer-layouts and goes into `/Users/cewen/code/consumer-layouts`. Then on the next iteration, consumer-domain, it isn't going up a directory so it becomes `/users/cewen/code/consumer-layouts/consumer-domain`

By just storing the root directory outside, it gets set to `/Users/cewen/code` and goes directly from that every time.